### PR TITLE
Simpler way to remove assetic bundle

### PR DESCRIPTION
**Pull request by Webarkitekt**
You could use the composer command line to remove the assetic bundle.

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -117,14 +117,7 @@ var AppGenerator = yeoman.generators.Base.extend({
...
         var done = this.async();
-
-        var composerContents = this.readFileAsString('composer.json');
-        var composerParse = JSON.parse(composerContents);
-        delete composerParse.require['symfony/assetic-bundle'];
-        var data = JSON.stringify(composerParse, null, 4);
-
-        fse.deleteSync(this.templatePath('composer.json'));
-        fs.writeFileSync('composer.json', data);
+        exec('composer remove "symfony/assetic-bundle"');
```
Comment (bezoerb) [inline app/index.js:120]: You missed the non-global composer

**Comment by bezoerb** [comment]
i agree this looks quite cleaner than manually editing the `composer.json` but it also takes longer to execute.

**Comment by bezoerb** [comment]
thanks anyway ;)

File: app/index.js (hunk 1 above)
Comment (Webarkitekt) [inline app/index.js:120]: if composer is installed non-globally it could also be run into the symfony install directory

**Comment by Webarkitekt** [comment]
I don't think that it takes much longer to execute. But as you prefer ;)

**Pull request closed without merging**